docker-compose.yml

```diff
@@ -16,6 +17,7 @@ services:
     # build: backend
     command: bash docker-entrypoint.sh
     # command: tail -f /dev/null
+
```

Comment:
Can you elaborate why this empty line is necessary :)?

Reply:
I have no idea :(